public/style.css

```diff
@@ -37,3 +38,64 @@ h2 {
   margin: 5vh 15vw;
   text-align: center;
 }
+
+/* flex stuff   */
+
+figure img{
+  width: 200px;
+}
+.header{
+  border:1px solid black;
+  margin-top: 5px;
+  margin-bottom: 5px;
```

Comment:
Although it's true you can use shorthand for this, it's really a good idea not to. Diffs are much easier to track if you separate things out by line. This goes doubly so for `background` as using the shorthand there will set overrides you might not know about.